packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -78,6 +78,129 @@ export class ComponentShowcase implements m.ClassComponent {
             })
           }
         /> */}
+        <h1>Popover Menu</h1>
+        <div class="basic-gallery">
+          <CWPopoverMenu
```

Comment:
@jnaviask @gdjohnson @affordances 
do we like this syntax for filling out the menu, an array of objects? 
or should it be formatted more like: 
```
<CWPopoverMenu 
  trigger={}
>
   <MenuItem label='' icon='' />
   <MenuItem type='divider' />
<CWPopoverMenu />

```

Reply:
No strong preference for me. I'm not sure of best practices in tsx.